### ISSUE TITLE
When adding 3 Digital Ocean hosts , one of them stuck in "Installing Rancher agent" state  forever 

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#6709]
Rancher Version:

Steps to reproduce the problem:
Had "Default" environment as "cattle" .
Created another environment with "kubernetes" template.
Added 3  Digital Ocean hosts to this environment.
When adding 3 Digital Ocean hosts , one of them stuck in "Installing Rancher agent" state  forever.

Following are the logs of the rancher agent which is in "Exited" state for sometime now.
```
root@sangeekube1:~# docker ps -a
CONTAINER ID        IMAGE                       COMMAND                  CREATED             STATUS                      PORTS               NAMES
561758c0f8d8        rancher/agent:v1.1.0-rc15   "/run.sh http://104.1"   21 minutes ago      Exited (1) 20 minutes ago                       rancher-agent-bootstrap
root@sangeekube1:~# 
```
```
INFO: Running Agent Registration Process, CATTLE_URL=http://104.155.155.162:8080/v1
INFO: Checking for Docker version >= 1.6.0
INFO: Found Server version: 1.12.3
INFO: docker version: Client version: 1.6.0
INFO: docker version: Client API version: 1.18
INFO: docker version: Go version (client): go1.4.2
INFO: docker version: Git commit (client): 4749651
INFO: docker version: OS/Arch (client): linux/amd64
INFO: docker version: Server version: 1.12.3
INFO: docker version: Server API version: 1.24
INFO: docker version: Go version (server): go1.6.3
INFO: docker version: Git commit (server): 6b644ec
INFO: docker version: OS/Arch (server): linux/amd64
INFO: docker info: Containers: 1
INFO: docker info: Images: 1
INFO: docker info: Storage Driver: aufs
INFO: docker info: Root Dir: /var/lib/docker/aufs
INFO: docker info: Backing Filesystem: extfs
INFO: docker info: Dirs: 11
INFO: docker info: Dirperm1 Supported: true
INFO: docker info: Execution Driver:
INFO: docker info: Kernel Version: 4.4.0-47-generic
INFO: docker info: Operating System: Ubuntu 16.04.1 LTS
INFO: docker info: CPUs: 1
INFO: docker info: Total Memory: 992.5 MiB
INFO: docker info: Name: sangeekube1
INFO: docker info: ID: DGWZ:B5SA:HOWM:2BMH:FQHE:3ASK:MT4A:A3X4:G47A:ATBP:3YSE:VDSC
INFO: docker info: Http Proxy:
INFO: docker info: Https Proxy:
INFO: docker info: No Proxy:
WARNING: No swap limit support
INFO: docker info: Labels:
INFO: docker info: provider=digitalocean
INFO: Attempting to connect to: http://104.155.155.162:8080/v1
INFO: http://104.155.155.162:8080/v1 is accessible
INFO: Inspecting host capabilities
INFO: Systemd: true
INFO: Boot2Docker: false
INFO: Host writable: true
INFO: Token: xxxxxxxx
INFO: Running registration
Traceback (most recent call last):
  File "./register.py", line 31, in <module>
    r = client.wait_success(r)
  File "/usr/local/lib/python2.7/dist-packages/cattle.py", line 15, in wait_success
    obj = self.wait_transitioning(obj, timeout)
  File "/usr/local/lib/python2.7/dist-packages/cattle.py", line 33, in wait_transitioning
    raise Exception(msg)
Exception: Timeout waiting for [register:1r6] to be done after 45.6967909336 seconds
root@sangeekube1:~# 
```

**Comment by StrongMonkey** [comment]
This happens when agent timeout in getting registration token from cattle. @ibuildthecloud is going to increase the timeout for getting registration token.

**Comment by deniseschannon** [comment]
Fixed in rancher/agent:v1.1.0-rc16, which will be available with v1.2.0-pre4-rc11

**Comment by sangeethah** [comment]
Tested with v1.2.0-pre4-rc11
Have not seen the issue where host is stuck in  "Installing Rancher agent" state " when adding multiple Digital Ocean hosts to this environment.
